src/sentry/api/endpoints/integrations/sentry_apps/internal_app_token/details.py

```diff
@@ -70,3 +71,28 @@ def delete(self, request: Request, sentry_app, api_token) -> Response:
         )
 
         return Response(status=204)
+
+
+# TODO: when the above endpoint is deprecated, simply move the convert method.
+# This endpoint handler utilizes the id instead.
+@control_silo_endpoint
+class NewSentryInternalAppTokenDetailsEndpoint(SentryInternalAppTokenDetailsEndpoint):
```

Comment:
will we rename this to remove the `new` once we've deprecated the old class?

can we instead rename the old one to `deprecated` or `old`?

Reply:
oh wait, nvm i see your ocmment

Reply:
No, the opposite actually! This entire class will be deleted. We just need an intermediary right now to help the FE get off using token values. Once the FE is updated, this "new" class will be deleted, and the old class will just get updated with the new logic. I want to avoid calling it "old" because it's not necessarily old since we'll be keeping it around